<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Warp Developement_Contact Us" duration-ms="28095" started-at="2019-04-10T14:23:32Z" finished-at="2019-04-10T14:24:00Z">
    <groups>
    </groups>
    <test name="Contact US Test" duration-ms="28095" started-at="2019-04-10T14:23:32Z" finished-at="2019-04-10T14:24:00Z">
      <class name="com.warpDevelopment.testCases.SubmitCorrectDetails_WD_TC_01">
        <test-method status="PASS" signature="initialization(java.lang.String)[pri:0, instance:com.warpDevelopment.testCases.SubmitCorrectDetails_WD_TC_01@8bd1b6a]" name="initialization" is-config="true" duration-ms="4320" started-at="2019-04-10T16:23:42Z" finished-at="2019-04-10T16:23:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="FAIL" signature="SubmitCorrectDetails__WD_TC_01()[pri:0, instance:com.warpDevelopment.testCases.SubmitCorrectDetails_WD_TC_01@8bd1b6a]" name="SubmitCorrectDetails__WD_TC_01" duration-ms="4548" started-at="2019-04-10T16:23:46Z" finished-at="2019-04-10T16:23:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.warpDevelopment.testCases.SubmitCorrectDetails_WD_TC_01.SubmitCorrectDetails__WD_TC_01(SubmitCorrectDetails_WD_TC_01.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitCorrectDetails__WD_TC_01 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.warpDevelopment.testCases.SubmitCorrectDetails_WD_TC_01@8bd1b6a]" name="tearDown" is-config="true" duration-ms="599" started-at="2019-04-10T16:23:51Z" finished-at="2019-04-10T16:23:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.warpDevelopment.testCases.SubmitCorrectDetails_WD_TC_01 -->
      <class name="com.warpDevelopment.testCases.ValidatePageTest_WD_TC_00">
        <test-method status="PASS" signature="initialization(java.lang.String)[pri:0, instance:com.warpDevelopment.testCases.ValidatePageTest_WD_TC_00@55f3ddb1]" name="initialization" is-config="true" duration-ms="8744" started-at="2019-04-10T16:23:32Z" finished-at="2019-04-10T16:23:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="PASS" signature="ValidateSite_WD_TC_00()[pri:0, instance:com.warpDevelopment.testCases.ValidatePageTest_WD_TC_00@55f3ddb1]" name="ValidateSite_WD_TC_00" duration-ms="298" started-at="2019-04-10T16:23:41Z" finished-at="2019-04-10T16:23:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateSite_WD_TC_00 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.warpDevelopment.testCases.ValidatePageTest_WD_TC_00@55f3ddb1]" name="tearDown" is-config="true" duration-ms="598" started-at="2019-04-10T16:23:41Z" finished-at="2019-04-10T16:23:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.warpDevelopment.testCases.ValidatePageTest_WD_TC_00 -->
      <class name="com.warpDevelopment.testCases.MandatoryFieldsOnly_WD_02">
        <test-method status="PASS" signature="initialization(java.lang.String)[pri:0, instance:com.warpDevelopment.testCases.MandatoryFieldsOnly_WD_02@18be83e4]" name="initialization" is-config="true" duration-ms="4249" started-at="2019-04-10T16:23:51Z" finished-at="2019-04-10T16:23:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="FAIL" signature="SubmitMandatoryFieldsOnly_WD_02()[pri:0, instance:com.warpDevelopment.testCases.MandatoryFieldsOnly_WD_02@18be83e4]" name="SubmitMandatoryFieldsOnly_WD_02" duration-ms="3961" started-at="2019-04-10T16:23:56Z" finished-at="2019-04-10T16:24:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.warpDevelopment.testCases.MandatoryFieldsOnly_WD_02.SubmitMandatoryFieldsOnly_WD_02(MandatoryFieldsOnly_WD_02.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitMandatoryFieldsOnly_WD_02 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.warpDevelopment.testCases.MandatoryFieldsOnly_WD_02@18be83e4]" name="tearDown" is-config="true" duration-ms="594" started-at="2019-04-10T16:24:00Z" finished-at="2019-04-10T16:24:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.warpDevelopment.testCases.MandatoryFieldsOnly_WD_02 -->
    </test> <!-- Contact US Test -->
  </suite> <!-- Warp Developement_Contact Us -->
</testng-results>
